### PR TITLE
Add ELSH custom axis

### PR DESCRIPTION
**Pull request by vv-monsalve**
PR to follow up on the inclusion of the new custom axis `ESHP` https://github.com/googlefonts/axisregistry/issues/15

### CODE REVIEW

File: Lib/axisregistry/data/element_shape.textproto [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#Element Shape, based on https://github.com/rosettatype/handjet
+tag: "ESHP"
```
Comment (chrissimpkins) [inline Lib/axisregistry/data/element_shape.textproto:2]: Decision made to revise this to "ELSP" from "ESHP"

**Review by chrissimpkins — CHANGES_REQUESTED** [review]

File: Lib/axisregistry/data/element_shape.textproto (hunk 1 above)
Comment (chrissimpkins) [inline Lib/axisregistry/data/element_shape.textproto:2]:
```suggestion
tag: "ELSP"
```

**Review by chrissimpkins — CHANGES_REQUESTED** [review]

File: Lib/axisregistry/data/element_shape.textproto (hunk 1 above)
Comment (chrissimpkins) [reply Lib/axisregistry/data/element_shape.textproto]: Addressed in https://github.com/googlefonts/axisregistry/pull/56/commits/aa483aa61e89b1f79079119cf239f6c4dc3a7047

**Comment by davelab6** [comment]
This needs images before merging, and description check with Sarah.

**Comment by vv-monsalve** [comment]
Display name + Tag under revision #78

**Comment by chrissimpkins** [comment]
https://github.com/googlefonts/axisregistry/pull/78#issuecomment-1261142617

**Review by chrissimpkins — APPROVED** [review]

**Comment by chrissimpkins** [comment]
Merged to google/fonts main in https://github.com/google/fonts/pull/5333

**Comment by chrissimpkins** [comment]
Let's move this axis registration to production so that we can prepare for the Handjet sandbox push.

https://github.com/google/fonts/pull/5426#discussion_r997718157

**Pull request merged**